apps/frontend/.env.example

```
@@ -0,0 +1,13 @@
+# Firebase Client SDK
+NEXT_PUBLIC_FIREBASE_API_KEY=AIzaSyATM5BPZL9oNYfxuuuE7phGYJkqMYictnM
+NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=safetrustcr-596e3.firebaseapp.com
+NEXT_PUBLIC_FIREBASE_PROJECT_ID=safetrustcr-596e3
+NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=safetrustcr-596e3.firebasestorage.app
+NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=736891312580
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID key should go before the NEXT_PUBLIC_FIREBASE_PROJECT_ID key

(UnorderedKey)